image-viewer.py

```diff
@@ -56,72 +59,91 @@ def render(self):
 
         self.root.bind("<KeyPress>", self.onKeyPress)
         self.root.mainloop()
-        
+
     def create_menu(self):
         menubar = Menu(self.root)
         self.root.config(menu=menubar)
-        
+
         file_menu = Menu(menubar, tearoff=False)
-        file_menu.add_command(label='Open Folder', command=self.open_folder, underline=0)
-        file_menu.add_command(label='Refresh', command=self.refresh, underline=0)
-        file_menu.add_command(label='Exit', command=self.root.destroy, underline=1)
-        menubar.add_cascade(label='File', menu=file_menu, underline=0)
-    
+        file_menu.add_command(label="Open File(s)", command=self.open_files)
+        file_menu.add_command(label="Open Folder", command=self.open_folder, underline=0)
+        file_menu.add_command(label="Refresh", command=self.refresh, underline=0)
+        file_menu.add_command(label="Exit", command=self.root.destroy, underline=1)
+        menubar.add_cascade(label="File", menu=file_menu, underline=0)
+
+    def open_files(self):
+        files = filedialog.askopenfilenames(
+            filetypes=(
+                ("All files accepted", ["*.png", "*.jpg", "*.jpeg", "*.ico", "*.gif"]),
+                ("JPEG", ["*.jpg", "*.jpeg"]),
+                ("GIF", "*.gif"),
+                ("PNG", "*.png"),
+                ("ICO", "*.ico"),
+            )
+        )
+
+        if len(files) == 0: # User cancelled
```

Comment:
This is a good use of a comment on line 85! It might not be obvious why the length of the files array might be 0 based on `if len(files) == 0`, so putting that comment in was a really good idea!